drivers/docker/driver.go

```diff
@@ -706,8 +706,13 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		hostConfig.MemorySwap = task.Resources.LinuxResources.MemoryLimitBytes // MemorySwap is memory + swap.
 	}
 
+	loggingDriver := driverConfig.Logging.Driver
+	if loggingDriver == "" {
+		loggingDriver = driverConfig.Logging.Type
```

Comment:
`Type` should take precedence as it was the only one that worked in pre-0.9. `Driver` has never had any effect so we need to make sure we don't turn what was a noop into an override. (I can't imagine anyone actually defines *both*, but I'd hate to learn the hard way.)

Reply:
Makes sense - I made type take precedence for now until we decide to deprecate it.